packages/schematics/angular/config/index.ts

```diff
@@ -103,3 +106,53 @@ function addKarmaConfig(options: ConfigOptions): Rule {
     );
   });
 }
+
+function addAiContextFile(options: ConfigOptions): Rule {
+  let fileName: string;
+  let filePath: string;
+  switch (options.tool) {
+    case 'gemini':
+      fileName = 'GEMINI.md';
+      filePath = '.gemini';
+      break;
+    case 'copilot':
+      fileName = 'copilot-instructions.md';
+      filePath = '.github';
+      break;
+    case 'claude':
+      fileName = 'CLAUDE.md';
+      filePath = '.claude';
+      break;
+    case 'cursor':
+      fileName = 'cursor.md';
+      filePath = path.join('.cursor', 'rules');
+      break;
+    case 'windsurf':
+      fileName = 'guidelines.md';
+      filePath = path.join('.windsurf', 'rules');
+      break;
+    default:
+      throw new SchematicsException(`Unsupported AI tool: ${options.tool}`);
+  }
+
+  return async (host) => {
+    const workspace = await readWorkspace(host);
+    const project = workspace.projects.get(options.project);
+    if (!project) {
+      throw new SchematicsException(`Project name "${options.project}" doesn't not exist.`);
+    }
+
+    // Keep this file in sync with the one presented here https://angular.dev/ai/develop-with-ai
+    // The file is located in the framework repo at adev/src/context/best-practices.md
+    const templateName = 'best-practices.md';
+
+    return mergeWith(
+      apply(url('./files'), [
+        filter((p) => p.endsWith(`${templateName}.template`)),
+        applyTemplates({}),
+        move(filePath),
+        rename(filePath, templateName, fileName),
```

Comment:
The rename and filtering can be removed, instead using applyTemplates to change the name.

```suggestion
        applyTemplates({ rulesName, directoryName }),
        move(filePath),
```

```
---files 
    -- __directoryName__
           --- __rulesName__.md.template
````